cv_next/lib/utils.ts

```diff
@@ -1,27 +1,27 @@
 import { type ClassValue, clsx } from "clsx"
```

Comment:
This file changes means there are conflicting prettier configs!

Reply:
Maybe that's tee semi false?